test/live_view_demo/white_list_test.exs

```diff
@@ -0,0 +1,27 @@
+defmodule LiveViewDemo.WhiteListTest do
+  use ExUnit.Case
+  alias LiveViewDemo.WhiteList
+
+  describe "validate/2" do
```

Comment:
I'm not using `describe` a lot, but this is inspiring me to start using it more often in this project